Source/WebCore/platform/graphics/texmap/TextureMapperFPSCounter.cpp

```diff
@@ -48,18 +49,18 @@ TextureMapperFPSCounter::TextureMapperFPSCounter()
 
 void TextureMapperFPSCounter::updateFPSAndDisplay(TextureMapper& textureMapper, const FloatPoint& location, const TransformationMatrix& matrix)
 {
-    if (!m_isShowingFPS)
```

Comment:
Not a hard blocker, I am only wondering: Could we also return early and avoid the FPS calculation when sysprof collection is disabled?

Reply:
Yup. Done!